keel-sql/src/main/resources/db/changelog/20200127-environment-artifact-pin-table.yml

```diff
@@ -0,0 +1,56 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-environment-artifact-pin-table
+      author: asher
+      changes:
+        - createTable:
+            tableName: environment_artifact_pin
+            columns:
+              - column:
+                  name: environment_uid
+                  type: char(26)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: artifact_uid
+                  type: char(26)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: artifact_version
+                  type: varchar(255)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: pinned_at
+                  type: bigint
```

Comment:
would this be better as a `timestamp` like other columns in our schema?

Reply:
The `unhappy_veto` table introduced millisecond epoch timestamps and there was some discussion around time types then. We should have consistency but I'd rather see all of the `timestamp` columns move to epoch like other spinnaker services. I think it's problematic that the `timezone` type is zoned (and the mysql running on our laptops is likely running in local time while our aurora instances are in UTC) and we're better off avoiding use of sql datetime functions that that the `timezone` type enables. Let's discuss offline though, if we don't want to go thru the pain of migrating existing columns, it does make sense to change the type here.

Reply:
Yeah, let's tackle it as a separate change if we already have inconsistency.